src/main/io/osd.h

```diff
@@ -101,6 +105,8 @@ typedef enum {
     OSD_FLIP_ARROW,
     OSD_LINK_QUALITY,
     OSD_TOTAL_DIST,
+    OSD_STICK_OVERLAY_LEFT,
+    OSD_STICK_OVERLAY_RIGHT,
```

Comment:
We need to increment the version of the `osdConfig` parameter group here, as we are increasing `OSD_ITEM_COUNT`. See https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md

Reply:
Oops.  Thanks, now fixed.